lib/hammer_cli_katello/repository.rb

```diff
@@ -255,6 +262,27 @@ def import_upload_ids(ids)
         }
         resource.call(:import_uploads, params)
       end
+
+      def progress_message(filename)
+        _('Uploading %{filename}') % {:filename => filename}
+      end
+
+      def render_result
+        print_message _('finished')
```

Comment:
I don't think what I am asking is all that much, just to check/ensure that if there is an error state the user is presented with that information.